debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/ServerInfo.java

```diff
@@ -156,14 +162,26 @@ protected boolean active() {
         protected Long latestFlushedLSN() {
             return latestFlushedLSN;
         }
+
+        protected Long restartLSN() {
+            return restartLSN;
```

Comment:
Is there a JavaDoc of the semantics of "restartLsn" vs. "latestFlushedLsn" somewhere? Doesn't have to be repeated in every field/getter representing it, but should be there once, in the "most natural" place.